mgmt_cli_test.py

```diff
@@ -31,7 +31,9 @@ class MgmtCliTest(ClusterTester):
 
     :avocado: enable
     """
-    MANAGER_IDENTITY_FILE = '/tmp/scylla_manager_pem'
+    MANAGER_IDENTITY_FILE_DIR = '/root/.ssh'
```

Comment:
You defined it in `mgmt.py`, why did you copy-paste it here?

Reply:
removed duplicate.